src/EditorFeatures/Core/GoToDefinition/GoToDefinitionHelpers.cs

```diff
@@ -129,5 +129,21 @@ internal static class GoToDefinitionHelpers
                 streamingPresenter.TryNavigateToOrPresentItemsAsync(
                     threadingContext, solution.Workspace, title, definitions, cancellationToken));
         }
+
+        public static bool TryGoToDefinition(
```

Comment:
💡 After this merges, can remove the previous `TryGoToDefinition` method as a separate follow-up. You have the option of either filing a follow-up issue or submitting a PR to just remove it in favor of the new method.